_pages/decide/task-flow-analysis.md

```diff
@@ -6,27 +6,31 @@ redirect_from: /task-flow-analysis/
 
 ## What it is
 
-A step-by-step analysis of a common task a user must perform that diagrams the various touch points and decision points a user goes through to accomplish the task. The touch points should be represented as steps taken by the user, as well as steps taken by the system.
+A step-by-step analysis of how a user will interact with a system in order to reach a goal. This analysis is documented in a diagram that traces a user's possible paths through sequences of tasks and decision points in pursuit of their goal. The tasks and decision points should represent steps taken by the user, as well as steps taken by the system.
 
 ## Reasons to use it
 
-To illustrate in a solution-agnostic way the overall flow that a user progresses through to accomplish a single task. Task flow analysis also demonstrates the relationship between tasks, and how they interconnect across a site.
+To validate a design team's understanding of users' goals, common scenarios, and tasks, and to illustrate in a solution-agnostic way the overall flow of tasks through which a user progresses to accomplish a goal.  Task flow diagrams also help surface obstacles in the way of users achieving their goal. 
+
+## Prerequisites
+
+**User research.** Task flow analysis requires that you have a baseline understanding of your users' knowledge, goals, and behaviors. 
 
 ## Time required
 
-**Medium:** 1-2 hours per task
+**Medium:** 2-3 hours per user goal
 
 ## How to do it
 
-1. Identify the task(s) that need to be analyzed.
+1. Based on user research, identify target users' goals that need to be analyzed.
 
-2. Break each high-level task down into the subtasks and decisions that the user or system must perform. Specify the subtask in terms of objectives. Across all subtasks, you should cover the whole area of interest. Don’t make assumptions about which steps are understood.
+2. For each goal, identify common scenarios and the tasks and decisions that the user or system will perform in each scenario. Don’t make assumptions about which steps are understood.
 
-3. Produce a layered task diagram of each subtask and decision point. The diagram must cover each step or decision necessary to accomplish the task.
+3. Produce a taskflow diagram that includes each task and decision point that a user might encounter on their way toward their goal. The diagram should cover the common scenarios identified in step 2.  
```

Comment:
I feel like "taskflow diagram" is right, but still jargon. Could we link to an example inline?